packages/strapi-generate-model/templates/bookshelf/model.settings.template

```diff
@@ -8,7 +8,11 @@
   "options": {
     "increments": true,
     "timestamps": true,
-    "comment": ""
+    "comment": "",
+    "plugins": [
```

Comment:
We don't need this because the visibility and pagination plugins are added automatically.